pubpub/pubpub
# feature: User-selectable CC licenses

This PR adds the ability to select a license for a Pub that will be displayed in its footer. I've added a button for this in the Pub footer as well as the Pub management page.

_Test plan:_
- Make sure that you can change the license from the pub manage > details pane
- Make sure that as a pub manager you can change the license from the pub-bottom footer
- Make sure you _cannot_ change the license from the footer if you are not a pub manager

_Screenshots:_
In the Pub management page:
<img width="855" alt="Screen Shot 2019-09-03 at 3 41 44 PM" src="https://user-images.githubusercontent.com/2208769/64203445-6bf6d980-ce61-11e9-8b8e-294533db4288.png">
Inline with the Pub footer:
<img width="612" alt="Screen Shot 2019-09-03 at 3 41 33 PM" src="https://user-images.githubusercontent.com/2208769/64203446-6bf6d980-ce61-11e9-8053-f65b3abd6a0e.png">
